CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Consistently raise ValueError instead of LookupErrors in places where a value is being removed.
+
+### Fixed
+- Actually call Command.bind_client and bind_component in the component add command methods and
+  specifically SlashCommand.bind_client in Component.bind_client.
+- Return the command object from Component.with_command methods.
```

Comment:
Remove when not present test coverage for components and clients + add when already present test coverage for client